src/payment.coffee

```diff
@@ -131,44 +127,81 @@ hasTextSelected = (target) ->
 
 reFormatCardNumber = (e) ->
   setTimeout =>
-    target = e.target
+    target  = e.target
     value   = QJ.val(target)
-    value   = Payment.fns.formatCardNumber(value)
-    QJ.val(target, value)
+
+    # clear the value
+    QJ.val target, ''
```

Comment:
What's the advantage of typing it out like this? I'm hesitant to add in all of these browser specific code as it feels like it makes it more fragile.